src/biome/text/pipeline.py

```diff
@@ -4,42 +4,54 @@
 import os
 import shutil
 import tempfile
+from biome.text import vocabulary
+from biome.text.configuration import FindLRConfiguration
+from biome.text.configuration import PipelineConfiguration
+from biome.text.configuration import TrainerConfiguration
+from biome.text.configuration import VocabularyConfiguration
+from biome.text.data import InstancesDataset
+from biome.text.dataset import Dataset
+from biome.text.errors import ActionNotSupportedError
+from biome.text.errors import EmptyVocabError
+from biome.text.features import TransformersFeatures
+from biome.text.helpers import update_method_signature
 from inspect import Parameter
 from pathlib import Path
-from typing import Any, Dict, Iterable, List, Optional, Type, Union, cast
+from typing import Any
+from typing import cast
+from typing import Dict
+from typing import Iterable
+from typing import List
+from typing import Optional
+from typing import Type
+from typing import Union
 
 import numpy
 import torch
 from allennlp.commands.find_learning_rate import search_learning_rate
 from allennlp.common import Params
-from allennlp.data import AllennlpDataset, AllennlpLazyDataset, Instance, Vocabulary
+from allennlp.data import AllennlpDataset
```

Comment:
Do we still need this import?

Reply:
No, I think it is useless. I have erased it.